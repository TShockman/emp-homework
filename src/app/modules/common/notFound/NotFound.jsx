import React, {PureComponent} from 'react';
import style from './notFound.scss';

export default class NotFound extends PureComponent {

  render() {
    return <div className="notFound"><h1>The page you were looking for could not be found!</h1></div>
  }
}