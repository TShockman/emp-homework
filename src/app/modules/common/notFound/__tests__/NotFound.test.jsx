import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import NotFound from '../';

const renderer = new ReactShallowRenderer();

describe('NotFound', () => {
  it('Renders consistently', () => {
    expect(renderer.render(<NotFound/>)).toMatchSnapshot();
  });
});