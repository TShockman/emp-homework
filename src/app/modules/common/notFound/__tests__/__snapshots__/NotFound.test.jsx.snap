// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Renders consistently 1`] = `
<div
  className="notFound"
>
  <h1>
    The page you were looking for could not be found!
  </h1>
</div>
`;
